### DISCUSSION TITLE
How to get this extension working with SD.next (vlad1111)

### DISCUSSION THREAD

**Discussion by mald55** [picobyte/stable-diffusion-webui-wd14-tagger#55] [Q&A]
Hello,

Can anyone help me to get this extension working with SD.next (vlad1111)?

It seems to install but it won't show up on the UI, any ideas?

**Comment by Nem-z** [comment]
For more info here is the error message:

![image](https://github.com/picobyte/stable-diffusion-webui-wd14-tagger/assets/63961359/37383ce9-b36e-4194-951a-c60803348bc2)

Tested on commit `f5831a117f9eadf085df8b44ab90a32c0b462501`
